morello/morello_pdf_conf.yaml

```diff
@@ -0,0 +1,16 @@
+title: Morello Supplement to the Arm C Language Extensions
+subtitle: 01alpha
+date: 02 July 2021
+logo: ./tools/Arm-logo-blue-RGB.pdf
+documentclass: report
+toc: true
+numbersections: true
+geometry: margin=2cm
+fontfamily: helvet
+classoption: 12pt
+fontcolor: darkgray
+colorlinks: citecolor=darkgray
+footertext: Copyright © 2020-2021, Arm Limited and its affiliates. All rights reserved.
```

Comment:
The text of the footer includes the page number. Please rename this variable to `copyright_text`.

Same for the other YAML header file.